### PR TITLE
New: (Indexer) Anidub

### PR DESCRIPTION
**Pull request by dm3ch**
#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

**Comment by dm3ch** [comment]
PR is ready for review.

This indexer is based on https://github.com/Jackett/Jackett/blob/6a213c6eabd0264392c48564ff16a9035fd13ca4/src/Jackett.Common/Indexers/AniDUB.cs#L315 code, and some parts was copied as is. Should we add some references to original project?

File: src/NzbDrone.Core/Indexers/Definitions/Anidub.cs [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+{
+    public class Anidub : TorrentIndexerBase<AnidubSettings>
+    {
+        public override string Name => "Anidub";
+        public override string[] IndexerUrls => new string[] { "https://tr.anidub.com/" };
+        public override string Description => "Anidub is russian anime voiceover group and eponymous anime tracker.";
+        public override string Language => "ru-ru";
+        public override Encoding Encoding => Encoding.UTF8;
+        public override DownloadProtocol Protocol => DownloadProtocol.Torrent;
+        public override IndexerPrivacy Privacy => IndexerPrivacy.Public;
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Definitions/Anidub.cs:33]: Given this requires auth is it really public?

Comment (dm3ch) [reply src/NzbDrone.Core/Indexers/Definitions/Anidub.cs]: Nice catch, Copy-paste error. This tracker is semi-public. Fixed that.

File: src/NzbDrone.Core/Indexers/Definitions/Anidub.cs [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+                {
+                    BookSearchParam.Q
+                },
+                MusicSearchParams = new List<MusicSearchParam>
+                {
+                    MusicSearchParam.Q
+                }
+            };
+
+            caps.Categories.AddCategoryMapping(2, NewznabStandardCategory.TVAnime, "Аниме TV");
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Definitions/Anidub.cs:123]: It probably makes sense to order these, I'm good otherwise on this PR

Comment (dm3ch) [reply src/NzbDrone.Core/Indexers/Definitions/Anidub.cs:123]: @Qstick Fixed

**Review by Qstick — APPROVED** [review]

**Comment by dm3ch** [comment]
@Qstick Here's pipeline failure that seems not actually caused by code. How can I trigger rerun of pipeline?

**Pull request merged**